packages/grafana-ui/src/components/PanelChrome/PanelChrome.test.tsx

```diff
@@ -18,6 +19,27 @@ const setup = (propOverrides?: Partial<PanelChromeProps>) => {
   return render(<PanelChrome {...props} />);
 };
 
+const setupWithToggle = (propOverrides?: Partial<PanelChromeProps>) => {
+  const props: PanelChromeProps = {
+    width: 100,
+    height: 100,
+    children: (innerWidth, innerHeight) => {
+      return <div style={{ width: innerWidth, height: innerHeight, color: 'pink' }}>Panel&apos;s Content</div>;
+    },
+    collapsible: true,
+  };
+
+  Object.assign(props, propOverrides);
+
+  const ControlledCollapseComponent = () => {
+    const [collapsed, toggleCollapsed] = useToggle(false);
+
+    return <PanelChrome {...props} collapsed={collapsed} onToggleCollapse={toggleCollapsed} />;
+  };
+
+  return render(<ControlledCollapseComponent />);
+};
+
```

Comment:
I think we should have two cases covered here: Controlled and uncontrolled

```jsx
// Controlled
<PanelChrome collapsible collapsed={collapsed} onToggleCollapse={toggleCollapsed} />

// Uncontrolled
<PanelChrome collapsible />
```
And both should collapse properly